game/scripts/vscripts/components/creeps/creep_power.lua

```diff
@@ -5,6 +5,6 @@ CreepPowerTable = {
   --  LEVEL     HEALTH    MANA      DAMAGE    ARMOR     GOLD      EXP
   {   0,        0.0,      0.0,      0.0,      0.0,      0.0,      0.0},
   {   1,        1.0,      1.0,      1.0,      1.0,      1.0,      1.0},
-  {   3,        1.2,      1.0,      1.0,      1.0,      1.2,      1.2},
-  {   1000,     100.0,    100.0,    100.0,   100.0,     100.0,    100.0}
+  {   3,        1.2,      1.0,      1.0,      1.0,      2.0,      1.2},
```

Comment:
Remove this line, it is not needed... This table is just an example how complex creep upgrade system is possible...

What this actually should contain is only rows number 2 and 4

Remove Level 0 and 3, these are not needed